CRM/Core/BAO/CustomGroup.php

```diff
@@ -358,10 +358,17 @@ public static function &getTree(
     if ($entityID) {
       $entityID = CRM_Utils_Type::escape($entityID, 'Integer');
     }
+    if (!is_array($subTypes)) {
+      if (empty($subTypes)) {
+        $subTypes = array();
+      }
+      else {
+        $subTypes = explode(',', $subTypes);
```

Comment:
Could the contact sybType name ever contain a comma? (ie, 'Bank, Regional' and 'Bank, National')
If so, then we need to impose the array format and/or CRM_Core_DAO::VALUE_SEPARATOR multi-value style